m2m/m2m.py

```diff
@@ -147,75 +148,74 @@ def writeTemplateFiles(self, baseDirectory, foldername):
         writeDirectory = os.path.join(baseDirectory, foldername)
         try:
             os.makedirs(writeDirectory)
-        except:
+        except OSError:
             if os.path.exists(writeDirectory):
                 pass  # no big deal if they exist
             else:
                 raise MetadataConverterException(
-                    "Unable to create the output directory '%s'. " +
-                    "Perhaps you should check permissions?" %
-                    (writeDirectory,))
-        with open(os.path.join(writeDirectory, "metadata.xml"), "wb") as templateFile:
+                    'Unable to create the output directory %s. ' +
+                    'Perhaps you should check permissions?' %
+                    writeDirectory)
+        with open(os.path.join(writeDirectory, 'metadata.xml'), 'wb') as templateFile:
             templateFile.write(self.__bytes__())
-        return "%s finished" % foldername
+        return '%s finished' % foldername
 
     def writeJSONFile(self, baseDirectory, foldername, data):
         baseDirectory = baseDirectory
         foldername = foldername
         writeDirectory = os.path.join(baseDirectory, foldername)
         try:
             os.makedirs(writeDirectory)
-        except:
+        except OSError:
             if os.path.exists(writeDirectory):
                 pass  # no big deal if they exist
             else:
                 raise MetadataConverterException(
-                    "Unable to create the output directory '%s'. " +
-                    "Perhaps you should check permissions?" %
-                    (writeDirectory,))
-        jsonFilename = open(os.path.join(writeDirectory, "metadata.json"), "w")
+                    'Unable to create the output directory %s.'
+                    'Perhaps you should check permissions?' % writeDirectory)
+        jsonFilename = open(os.path.join(writeDirectory, 'metadata.json'), 'w')
         jsonFile = json.dumps(data,
                               sort_keys=True,
                               indent=4,
                               separators=(',', ': ')
                               )
         jsonFilename.write(jsonFile)
         jsonFilename.close()
-        return "%s finished JSON" % foldername
+        return '%s finished JSON' % foldername
 
 
-if "__main__" == __name__:
+if '__main__' == __name__:
```

Comment:
We could reorder this like you did other variable comparisons: `if __name__ == '__main__':`

Reply:
oh..yea. I missed this one. Will change it.